release-team/role-handbooks/ci-signal/README.md

```diff
@@ -172,6 +184,18 @@ For more detailed information about what's on these dashboards, see [Release Blo
 
 The appropriate response to a failing or flaking job on either a blocking or informing dashboard is to open an issue. The primary goal of opening an issue is to track the work that needs to be done to bring the job back to a healthy status while providing accurate signal. The format for a helpful and informative issue may vary depending on the type of job, the test(s) that are failing for the job, and the responsible parties for the job.
 
+Before creating a new github issue, check if somebody else has already opened an issue. To filter already opened issues, [queries](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-project-board) can be applied.
+
+Example queries:
+
+```bash
```

Comment:
Is it bash for this kind of snippet?

Reply:
Not really but it highlights it and improves readability (IMHO)